paywall/src/unlock.js/startup.ts

```diff
@@ -41,7 +40,24 @@ export function startup(
   constants: StartupConstants
 ) {
   // normalize all of the lock addresses
-  const config = normalizeConfig(window.unlockProtocolConfig)
+  let config = normalizeConfig(window.unlockProtocolConfig)
+
+  // There is no reason to do anything if window.web3 does not exist
+  // and the config does not allow for user accounts. As a quick hack,
+  // when that's the case we will purposely make the config invalid so
+  // that we don't make any requests for lock data.
+  const userAccountsAllowed = !!config.unlockUserAccounts
+  const web3Present = !!window.web3
+  if (!web3Present && !userAccountsAllowed) {
+    config = {
+      ...config,
+      // This violates the expected value for locks on the paywall,
+      // which will force the checkout into the "no wallet" state
+      // without ever querying for any locks.
+      locks: new XMLHttpRequest(),
```

Comment:
wierd. can we just set it to be `{}`?

Reply:
I figured that would be guaranteed to be invalid, but I can change it